packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_account_utxos_to_account.test.ts

```diff
@@ -0,0 +1,137 @@
+})
+
+afterAll(async () => {
+  await container.stop()
+})
+
+beforeEach(async () => {
+  await providers.randomizeEllipticPair()
+  await container.waitForWalletBalanceGTE(101)
+
+  // fund utxos balance
+  await fundEllipticPair(container, providers.elliptic.ellipticPair, 1.1) // 1.1
+  await fundEllipticPair(container, providers.elliptic.ellipticPair, 5.5) // 6.6
+  await fundEllipticPair(container, providers.elliptic.ellipticPair, 10.566) // 17.166
+  await fundEllipticPair(container, providers.elliptic.ellipticPair, 15.51345) // 32.67945
+  await fundEllipticPair(container, providers.elliptic.ellipticPair, 20) // 52.67945
+  await fundEllipticPair(container, providers.elliptic.ellipticPair, 37.98) // 90.65945
+  await fundEllipticPair(container, providers.elliptic.ellipticPair, 9.34055) // 100
+
+  await providers.setupMocks()
+})
+
+describe('account.utxosToAccount()', () => {
+  it('should receive token and change', async () => {
+    const destPubKey = await providers.ellipticPair.publicKey()
+    const dest = await providers.elliptic.script()
+
+    const conversionAmount = 12.34
+
+    const utxosToAccount: UtxosToAccount = {
+      to: [{
+        balances: [{
+          token: 0x00,
+          amount: new BigNumber(conversionAmount)
+        }],
+        script: dest
+      }]
+    }
+
+    const txn = await builder.account.utxosToAccount(utxosToAccount, dest)
+
+    const outs = await sendTransaction(container, txn)
+    const change = await findOut(outs, providers.elliptic.ellipticPair)
+
+    expect(outs.length).toEqual(2)
+    const encoded: string = OP_CODES.OP_DEFI_TX_UTXOS_TO_ACCOUNT(utxosToAccount).asBuffer().toString('hex')
+    // OP_RETURN + utxos full buffer
+    const expectedRedeemScript = `6a${encoded}`
+    expect(outs[0].value).toEqual(conversionAmount)
+    expect(outs[0].scriptPubKey.hex).toEqual(expectedRedeemScript)
+
+    expect(change.value).toBeLessThan(100 - conversionAmount)
+    expect(change.value).toBeGreaterThan(100 - conversionAmount - 0.001)
+    expect(change.scriptPubKey.hex).toBe(`0014${HASH160(destPubKey).toString('hex')}`)
+    expect(change.scriptPubKey.addresses[0]).toBe(Bech32.fromPubKey(destPubKey, 'bcrt'))
+
+    // minted token
+    const account = await jsonRpc.account.getAccount(await providers.getAddress())
+    expect(account.length).toStrictEqual(1)
+    expect(account).toContain('12.34000000@DFI')
+  })
+
+  it('should reject invalid utxosToAccount arg - more than 1 token balances object', async () => {
+    const dest = await providers.elliptic.script()
+    await expect(builder.account.utxosToAccount({
+      to: [{
+        balances: [{
+          token: 0x00,
+          amount: new BigNumber(10)
+        }],
+        script: dest
+      }, {
+        balances: [{
+          token: 0x00,
+          amount: new BigNumber(10)
+        }],
+        script: dest
+      }]
+    }, dest)).rejects.toThrow('Conversion output `utxosToAccount.to` array length must be one')
```

Comment:
This shouldn't reject, we can send multiple if different script destinations are provided

Reply:
For simplicity reason, keep to 1 now. We don't want to create complexity at the start.

Reply:
@ivan-zynesis we don't need multiple sending at the start, we definitely will need to add it in later. You mind creating a issue for it to be addressed in the future?

Reply:
other than create issue but also recommend add a `TODO()` here

Reply:
#270